pve2_api.class.php

```diff
@@ -165,7 +165,13 @@ private function action ($action_path, $http_method, $put_post_parameters = null
 		// Lets decide what type of action we are taking...
 		switch ($http_method) {
 			case "GET":
-				// Nothing extra to do.
+				// cURL used to translate POSTFIELDS into the query string when the
+				// request method was GET, but that doesn't seem to be the case any
+				// longer, so we need to build them into the query string ourselves.
+				$action_postfields_string = http_build_query($put_post_parameters);
+				$action_path .= (strpos($action_path, '?') === FALSE ? '?' : '&') . $action_postfields_string;
```

Comment:
Can do.  I agree that `? :` can be overused, but this is a prime example of the reason it exists in the first place, and I don't feel it decreases readability in this case, so that's why I used it.